### DISCUSSION TITLE
Ingame slash command api

### DISCUSSION THREAD

**Discussion by evokerking1** [eaglerforge/EaglerForge#8] [Ideas]
Can you try to develop a ingame slash command API to register and execute commands.?

**Comment by wxnnvs** [comment]
nice idea

**Comment by WilliamVanBeast** [comment]
@radmanplays 
add this shit

**Comment by evokerking1** [comment]
I think mojang uses brigadier to register commands so try to expose brigadier

**Comment by evokerking1** [comment]
https://github.com/Mojang/brigadier

Is brigadier

**Reply by radmanplays** [reply]
brigadier was added in 1.14

**Comment by leahanderson1** [comment]
> Can you try to develop a ingame slash command API to register and execute commands.?

you mean tab completion for commands?

**Reply by radmanplays** [reply]
that dosnt even exist in 1.8

**Reply by STPv22** [reply]
he means using ModAPI to make custom commands

**Comment by STPv22** [comment]
I very much think that this is a good idea

**Comment by Block2222** [comment]
OsmiumCore is gonna have a slash command api.

website: https://osmium.minecraft.pe/
discord: https://discord.gg/qfE5paTcba/

**Reply by wxnnvs** [reply]
Bro is advertising everywhere lol
